mlrun/feature_store/api.py

```diff
@@ -322,6 +323,17 @@ def _rename_source_dataframe_columns(df):
     return df
 
 
+def validate(featureset: Union[FeatureSet, str]):
+    entities_keys = [entity.name for entity in featureset.spec.entities]
+    for step_name, step_value in featureset.graph.steps.items():
+        if step_name == DropFeatures.__name__:
```

Comment:
Can be simplified a bit (no loop):
```suggestion
    if DropFeatures.__name__ in featureset.graph.steps:
        step = featureset.graph.steps[DropFeatures.__name__]
```
Could be even a bit simpler if it were a real dictionary and not an instance of `ObjectDict`.

Reply:
tnx! great idea